src/woo_publications/publications/tests/test_publicaiton_identifier_admin.py

```diff
@@ -0,0 +1,285 @@
+
+    def test_publicationidentifier_admin_update(self):
+        publicatie, publicatie_2 = PublicationFactory.create_batch(2)
+
+        publication_identifier = PublicationIdentifierFactory.create(
+            publicatie=publicatie,
+            kenmerk="openzaak",
+            bron="documents api",
+        )
+
+        response = self.app.get(
+            reverse(
+                "admin:publications_publicationidentifier_change",
+                kwargs={"object_id": publication_identifier.pk},
+            ),
+            user=self.user,
+        )
+
+        form = response.forms["publicationidentifier_form"]
+        form["publicatie"].select(text=str(publicatie_2))
+        form["kenmerk"] = "gpp-zoeken"
+        form["bron"] = "search api"
+
+        update_response = form.submit("_save")
+
+        self.assertEqual(update_response.status_code, 302)
+
+        publication_identifier.refresh_from_db()
+
+        self.assertEqual(publication_identifier.publicatie, publicatie_2)
+        self.assertEqual(publication_identifier.kenmerk, "gpp-zoeken")
+        self.assertEqual(publication_identifier.bron, "search api")
+
+    def test_publicationidentifier_admin_delete(self):
+        publication_identifier = PublicationIdentifierFactory.create(
+            kenmerk="openzaak",
+            bron="documents api",
+        )
+
+        response = self.app.get(
+            reverse(
+                "admin:publications_publicationidentifier_delete",
+                kwargs={"object_id": publication_identifier.pk},
+            ),
+            user=self.user,
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        form = response.forms[1]
+
+        with self.captureOnCommitCallbacks(execute=True):
+            response = form.submit()
+
+        self.assertEqual(response.status_code, 302)
+        self.assertFalse(
+            PublicationIdentifier.objects.filter(pk=publication_identifier.pk).exists()
+        )
+
+
+@disable_admin_mfa()
+class TestInlineInformationCategoryForPublicationAdmin(WebTest):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.user = UserFactory.create(superuser=True)
+
+    def test_inline_publicationidentifier_admin_create(self):
+        ic = InformationCategoryFactory.create()
+        publication = PublicationFactory.create(
+            informatie_categorieen=[ic],
+            officiele_titel="title one",
+        )
+        reverse_url = reverse(
+            "admin:publications_publication_change",
+            kwargs={"object_id": publication.id},
+        )
+
+        response = self.app.get(reverse_url, user=self.user)
+
+        self.assertEqual(response.status_code, 200)
+
+        form = response.forms["publication_form"]
+        form["publicationidentifier_set-TOTAL_FORMS"] = (
+            "1"  # we're adding one, dynamically
+        )
+        add_dynamic_field(form, "publicationidentifier_set-0-kenmerk", "openzaak")
```

Comment:
```suggestion
        add_dynamic_field(form, "publicationidentifier_set-0-kenmerk", "OZ-123")
```
let's use a slightly more realistic identifier - this would typically be a zaaknummer for example.